Clinical-Genomics/scout
# Better handling of delivery reports, especially for rerun cases

Fix #1888

**How to test**:

1. Modify the demo case, by changing its `analysis_date` to today: "2020-04-28 16:00:46.000"

1. Add a case_obj["analyses"] field, consisting of an array like this:

 - One analysis has the same report as in case_obj["delivery_report"], date: 2020-04-28 16:00:46.000
- One analysis is missing the report (delivery_report = None), date: 2018-04-21 08:30:25.000
- One analysis is very old and has an old delivery report, different than the newest. date: 2015-04-21 08:30:25.000

Example:


![image](https://user-images.githubusercontent.com/28093618/80495922-cef52d80-8968-11ea-9043-317b71a5f35d.png)


1. make sure that the case page side bar displays only:
- Most recent delivery report
- Very old delivery report from 2015




**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
